Patch for cgen/version.py to deal with +computecanada version tag

Author: Oliver Stueker
Date:   2025-07-21
--- a/cgen/version.py	2025-07-21 17:09:32.291154800 -0230
+++ b/cgen/version.py	2025-07-21 17:10:53.327582897 -0230
@@ -4,7 +4,7 @@
 def _parse_version(version: str) -> tuple[tuple[int, ...], str]:
     import re
 
-    m = re.match(r"^([0-9.]+)([a-z0-9]*?)$", VERSION_TEXT)
+    m = re.match(r"^([0-9.]+)(\+[a-z0-9]*?)$", VERSION_TEXT)
     assert m is not None
 
     return tuple(int(nr) for nr in m.group(1).split(".")), m.group(2)
